index.html

```diff
@@ -9,8 +9,10 @@ <h2> Here are the people in the class </h2>
 <li><a href="https://www.vijayp.ca/">Vijay Pandurangan</a> (coinstructor) </li>
 </ul>
 
-<h2> Sophia Wang </h2> 
-Hello everyone! This is my section. 
+<ul>
```

Comment:
Still not 100% right;  you don't need to create a whole new `<ul>`.
If you delete line 10 and 12 (i.e. remove the `</ul>` and the `<ul> ` next to one another) it will be better.

Basically the `<ul>` says "everything after this until `</ul>` is in one list. if you close (`<TAGNAME>` is considered open in html and `</TAGNAME>` is considered close and everything between them is considered inside one entity.